website/docs/reference/playground.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: The Playground
+---
+
+:::info Availability
+
+The Unleash playground is available from Unleash 4.14 onwards. The playground is available in all Unleash versions.
+
+:::
+
+![The Unleash Playground form and an indication of where in the nav menu it is located.](/img/playground-form.png)
+
+The unleash playground is a part of the Unleash Admin UI and an accompanying API. The playground lets you see which of your instance's feature toggles would be enabled for a given Unleash context. It has multiple uses, such as:
+
+-   Understanding how Unleash evaluates strategies, constraints, and segments.
+-   Debugging Unleash contexts and toggles behaving differently from what you expect.
+-   Tailoring a set of strategies for given contexts.
+
+Each feature toggle will contain info on whether it was enabled or not and which variant it was assigned. Later versions of the playground also contain [detailed evaluation results](#detailed-evaluation-results) to help you understand exactly why the feature was enabled or disabled.
+
+## Configuring the context
+
+This section describes what information the playground needs to evaluate the context against your existing features.
+
+### Environment and projects
+
+While technically not part of the Unleash context proper, the playground needs to know what environment to evaluate the toggles against, and it also requires a list of projects to evaluate feature toggles from.
+
+Features can only be evaluated against a single environment at a time. The environment should be one of the environments configured for your Unleash instance.
+
+The playground will only evaluate fetaures belonging to the projects you specify. The projects parameter can either be a list of projects **or** _all_ projects.
+
+### The Unleash context
+
+The [Unleash context](../user_guide/unleash-context.md) represents a client SDK's configuration and used for evaluating your features. You can construct your own context or use a JSON version of a context from a client.
+
+#### Required fields
+
+Unleash clients all require an `appName` to start up. If you do not provide an `appName`, the playground will assign a default value to this property instead.
```

Comment:
I've actually left this out on purpose in case we want to change it in the future. That said, I'm not necessarily against changing it. And at this point, we _could_ actually remove that requirement, as we don't use the actual sdk anymore.

What do you think?